exampleplugins/5.2/SwagDigitalPublishingSample/Subscriber/WidgetSubscriber.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace SwagDigitalPublishingSample\Subscriber;
+
+use Enlight\Event\SubscriberInterface;
+
+/**
+ * Class Resources
+ * @package Shopware\DigitalPublishingSample\Subscriber
+ */
+class WidgetSubscriber implements SubscriberInterface
+{
+    /**
+     * @var string
+     */
+    private $pluginBaseDirectory;
+
+    /**
+     * Subscriber class constructor.
+     *
+     * @param string $pluginBaseDirectory
+     */
+    public function __construct($pluginBaseDirectory)
+    {
+        /**
+         * Set the bootstrap class for later use.
+         */
+        $this->pluginBaseDirectory = $pluginBaseDirectory;
+    }
+
+    /**
+     * Returns an array of events you want to subscribe to
+     * and the names of the corresponding callback methods.
+     *
+     * @return array
+     */
+    public static function getSubscribedEvents()
+    {
+        return array(
+            'Enlight_Controller_Action_PostDispatchSecure_Widgets_SwagDigitalPublishing' => 'onPostDispatchFrontend',
+            'Enlight_Controller_Action_PostDispatchSecure_Widgets_Emotion' => 'onPostDispatchFrontend',
+        );
+    }
+
+    /**
+     * Adds the template directory of the plugin to extend the frontend templates.
+     *
+     * @param \Enlight_Event_EventArgs $args
+     */
+    public function onPostDispatchFrontend(\Enlight_Event_EventArgs $args)
```

Comment:
:sheep: mäh, there are only widgets events, but the callback method is called on Frontend :wink:

Reply:
D